app/models/gems_adoption.rb

```diff
@@ -1,10 +1,12 @@
 class GemsAdoption < ActiveRecord::Base
+  enum status: [ :open, :close ]
+
   belongs_to :user
   belongs_to :ruby_gem
   has_many :adoption_requests
   validates :ruby_gem, :user, presence: true
 
-  scope :search, -> (keyword) { includes(:ruby_gem).where("ruby_gems.name like ?", "%#{keyword}%").references(:ruby_gem) }
+  default_scope { where(status: 'open') }
 
-  #scope :search, -> (keyword) { includes(:ruby_gem).where("description like ?", "%#{keyword}%") }
```

Comment:
Please don't leave commented out code.], this will be confusing in the future and it's generally not useful.